Userland/Libraries/LibProtocol/Request.cpp

```diff
@@ -54,6 +54,40 @@ void Request::stream_into(OutputStream& stream)
     };
 }
 
+void Request::stream_into(Core::Stream::Stream& stream)
```

Comment:
It's unfortunate that `AK::OutputStream` is incompatible with `Core::Stream` classes, but its API is a little baroque and not TRY-friendly. Perhaps in the future we can rectify that.